### PR TITLE
Update hi.js (v 3.8.6)

### PR DESCRIPTION
**Pull request by akaRupture**
#179  Left line 104 and 109, also need someone to confirm line 652

### CODE REVIEW

File: hi.js [hunk 1]
```diff
@@ -97,17 +97,17 @@ settings.experimental.header आजमाइशी
...
-settings.client.discordRPC Discord RPC
-settings.client.resourceSwapper Resource Swapper
-settings.client.clearCache Clear Cache
-settings.client.resetSettings Reset Client Settings
-settings.client.acceleratedCanvas Accelerated Canvas
+settings.client.header Client क्लाइंट
+settings.client.unlimitedFrames Unlimited FPS असीमित FPS
+settings.client.gameCap Game Capture खेल कैप्चर
+settings.client.experimental Experimental Features आजमाइशी फ़ीचर
+settings.client.discordRPC Discord RPC
```
Comment (HooD-ii) [inline hi.js:104]: डिस्कॉर्ड आर पी सी

File: hi.js [hunk 2]
```diff
@@ -97,17 +97,17 @@ settings.experimental.header आजमाइशी
...
+settings.client.header Client क्लाइंट
+settings.client.unlimitedFrames Unlimited FPS असीमित FPS
+settings.client.gameCap Game Capture खेल कैप्चर
+settings.client.experimental Experimental Features आजमाइशी फ़ीचर
+settings.client.discordRPC Discord RPC 
+settings.client.resourceSwapper Resource Swapper साधन अदला-बदली
+settings.client.clearCache Clear Cache कैश साफ़
+settings.client.resetSettings Reset Client Settings रीसेट क्लाइंट सेटिंग
+settings.client.acceleratedCanvas Accelerated Canvas एक्सेलरेटेड कैन्वस
 settings.client.angleBackend ANGLE Graphics Backend
```
Comment (HooD-ii) [inline hi.js:109]: ANGLE ग्राफ़िक्स बैकेंड

File: hi.js [hunk 3]
```diff
@@ -649,7 +649,7 @@ server.config.private प्राइवेट
 server.config.krRewards के.आर के पुरस्कार
 server.config.forceDefault डिफ़ॉल्ट सेटिंग्स को बलपूर्वक लागू करे
 server.config.warmupTime वार्मअप का समय
-server.config.gamRounds Rounds
+server.config.gamRounds Rounds दौर
```
Comment (HooD-ii) [inline hi.js:652]: दौर this is OK i guess

File: hi.js [hunk 4]
```diff
@@ -97,17 +97,17 @@ settings.experimental.header आजमाइशी
...
-settings.client.clearCache Clear Cache
-settings.client.resetSettings Reset Client Settings
-settings.client.acceleratedCanvas Accelerated Canvas
+settings.client.header Client क्लाइंट
+settings.client.unlimitedFrames Unlimited FPS असीमित FPS
+settings.client.gameCap Game Capture खेल कैप्चर
+settings.client.experimental Experimental Features आजमाइशी फ़ीचर
+settings.client.discordRPC Discord RPC 
+settings.client.resourceSwapper Resource Swapper साधन अदला-बदली
+settings.client.clearCache Clear Cache कैश साफ़
```
Comment (HooD-ii) [inline hi.js:106]: preferred let cache be cache only Cuz ppl might confuse for CASH

File: hi.js [hunk 5]
```diff
@@ -789,6 +789,6 @@ social.trades.decline सभी ट्रेड्स को खारिज क
 social.trades.message कृपया सेटिंग से व्यापार संदेश सक्षम करें
 social.trades.warning आपको इस व्यापार से कोइ वस्तु नही मिलेगा!
 social.trades.accept क्या आप वास्तव में इस ट्रेड को स्वीकार करना चाहते हैं?
-stats.airdrops KR-Packages
-settings.quality.flapAnim Sniper Flap Animation
+stats.airdrops KR-Packages KR पैकेज
+settings.quality.flapAnim Sniper Flap Animation स्नाइपर फ्लैप एनीमेशन
```
[the remark this answers is not in the document]
Comment (akaRupture) [reply hi.js:793]: स्नाइपर is correct

File: hi.js [hunk 6]
```diff
@@ -97,17 +97,17 @@ settings.experimental.header आजमाइशी
...
-settings.client.angleBackend ANGLE Graphics Backend
-settings.client.colorProfile Color Profile
+settings.client.header Client क्लाइंट
+settings.client.unlimitedFrames Unlimited FPS असीमित FPS
+settings.client.gameCap Game Capture खेल कैप्चर
+settings.client.experimental Experimental Features आजमाइशी फ़ीचर
+settings.client.discordRPC Discord RPC डिस्कॉर्ड आर पी सी
+settings.client.resourceSwapper Resource Swapper साधन अदला-बदली
+settings.client.clearCache Clear Cache Cache साफ़
+settings.client.resetSettings Reset Client Settings रीसेट क्लाइंट सेटिंग
```
Comment (mty22) [inline hi.js:107]: @akaRupture can you please remove the English components where you don't need it?

Comment (mty22) [reply hi.js:107]: across all your changes etc.

**Review by mty22 — CHANGES_REQUESTED** [review]

File: hi.js (hunk 6 above)
Comment (akaRupture) [reply hi.js:107]: done

File: hi.js [hunk 7]
```diff
@@ -98,16 +98,16 @@ settings.experimental.aimFreezeFix लक्ष्य के दौरान फ
...
-settings.client.gameCap Game Capture खेल कैप्चर
-settings.client.experimental Experimental Features आजमाइशी फ़ीचर
-settings.client.discordRPC Discord RPC डिस्कॉर्ड आर पी सी
-settings.client.resourceSwapper Resource Swapper साधन अदला-बदली
-settings.client.clearCache Clear Cache कैश साफ़
-settings.client.resetSettings Reset Client Settings रीसेट क्लाइंट सेटिंग
-settings.client.acceleratedCanvas Accelerated Canvas एक्सेलरेटेड कैन्वस
-settings.client.angleBackend ANGLE Graphics Backend ऐंगल ग्राफ़िक्स बैकेंड
-settings.client.colorProfile Color Profile रंग प्रोफ़ाइल
+settings.client.unlimitedFramesअसीमित FPS
```
Comment (mty22) [inline hi.js:101]: just need a space inserted here

**Comment by mty22** [comment]
pending native speaker review (I've pinged in Krunker India discord)

**Review by HistoriFy — COMMENTED** [review]
101 ,104, 106 are fine.

Rest are either too technical or just not the "exact" meaning.

I hope you understand

**Comment by HistoriFy** [comment]
> 101 ,104, 106 are fine.
> 
> Rest are either too technical or just not the "exact" meaning.
> 
> I hope you understand

more we are not looking for terms which are "Half English" and "Half Hindi"

**Comment by akaRupture** [comment]
I understand what you are saying @HistoriFy but some words like 'client', 'cache', 'backend', 'profile', 'graphics', 'flap' etc. don't exist in the Hindi language(atleast not to my knowledge). If you know any, I'd be happy to incorporate them.

**Comment by HistoriFy** [comment]
> I understand what you are saying @HistoriFy but some words like 'client', 'cache', 'backend', 'profile', 'graphics', 'flap' etc. don't exist in the Hindi language(atleast not to my knowledge). If you know any, I'd be happy to incorporate them.

thats what I am trying to say.

Those terms are too technical for translation to basic Hindi. Even if translate em, it will be hard for them to understand.

So its better to keep it as simple as possible and leave the terms if we are not able to translate them

**Comment by mty22** [comment]
If it's not translatable, just leave it in English. :)

**Comment by mty22** [comment]
anything else blocking this merge? @HistoriFy

**Comment by akaRupture** [comment]
@mty22 should i change anything?

**Comment by HistoriFy** [comment]
> anything else blocking this merge? @HistoriFy

Ill ask zeude to re-confirm once again

**Comment by HistoriFy** [comment]
@mty22 lines reviewed, can be merged now

**Comment by HistoriFy** [comment]
sorry for the delay, was sick

**Comment by mty22** [comment]
@akaRupture @HooD-ii please tell me your Krunker IGNs

**Comment by HooD-ii** [comment]
> @akaRupture @HooD-ii please tell me your Krunker IGNs

Mine is HooD_ii

**Pull request merged**